### PR TITLE
Add Podspec file & Remove dependency of AppDelegate

### PR DESCRIPTION
**Pull request by JAStanton**
Remove dependency of AppDelegate, more on the dangers of that here: https://github.com/yamill/react-native-orientation/pull/66#issuecomment-225297251

Next I added `.podspec` file for anyone using coacoa pods.

Note special shoutout to @mcrumm for getting the ball rolling on this. Sorry I wasn't able to update your branch on your fork (git permissions issue), that and I went down a different route for my podspec. Hope you can forgive me!

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -8,9 +8,25 @@ Run `npm install react-native-orientation --save`
 
 #### iOS
 
+With Coacoa pods add:
+
+  ```ruby
+  pod 'ReactNativeOrientation'
```
Comment (nevir) [inline README.md]:
```
pod 'ReactNativeOrientation', :path => './node_modules/react-native-orientation'
```

File: README.md [hunk 2]
```diff
@@ -8,9 +8,25 @@ Run `npm install react-native-orientation --save`
...
+  ```
+
+Or manually by:
+
 1. Open your project in XCode, right click on your project and click `Add Files to "Your Project Name"`
 2. Add `RCTOrientation` folder from your `node_modules/react-native-orientation` folder. <b>Make sure you have 'Create Groups' selected</b>
 
+Next, in `AppDelegate.m` add:
+
+```objc
```
Comment (nevir) [inline README.md]: don't forget to mention importing it

File: ReactNativeOrientation.podspec [hunk 3]
```diff
@@ -0,0 +1,22 @@
+require 'json'
+
+package = JSON.parse(File.read(File.join(__dir__, 'package.json')))
+
+Pod::Spec.new do |s|
+  s.name           = 'ReactNativeOrientation'
+  s.version        = package["version"]
+  s.summary        = package["description"]
+  s.description    = package["description"]
+  s.license        = package['license']
```
Comment (nevir) [inline ReactNativeOrientation.podspec]: consistent quotes (single everywhere)

File: ReactNativeOrientation.podspec [hunk 4]
```diff
@@ -0,0 +1,22 @@
+require 'json'
+
+package = JSON.parse(File.read(File.join(__dir__, 'package.json')))
+
+Pod::Spec.new do |s|
+  s.name           = 'ReactNativeOrientation'
```
Comment (nevir) [inline ReactNativeOrientation.podspec]: For consistency with other RN packages, go with `RNOrientation`

**Comment by yamill** [comment]
thanks @nevir 👍

**Pull request closed without merging**